libseccomp/src/lib.rs

```diff
@@ -632,18 +632,21 @@ impl ScmpFilterContext {
         Ok(())
     }
 
-    /// add_rule adds a single rule for an unconditional or conditional action on a syscall.
+    /// Adds a single rule for an unconditional action on a syscall.
+    pub fn add_rule(&mut self, action: ScmpAction, syscall: i32) -> Result<()> {
+        self.add_rule_conditional(action, syscall, &[])
+    }
+
+    /// Adds a single rule for a conditional action on a syscall.
     ///
-    /// Accepts the number of the syscall the action and the conditions to be taken on the call being made.
-    /// If the compartors is None, the function adds a single rule for an unconditional action.
-    /// Returns an error if an issue was encountered adding the rule.
-    pub fn add_rule(
+    /// Comparators are AND'd together (i.e. all must match for the rule to match).
+    /// You can only compare each argument once in a single rule.
```

Comment:
Could you add the following description?

```
/// If the specified rule needs to be adjusted due to architecture specifics,
/// it will be adjusted without notification. 
```